TotalSupplyRange Simulation Param Change Fails

**Describe the bug**
A clear and concise description of what the bug is.

The RandomizedParamChange in the simulation package for the TotalSupplyRange param causes a fail when the simulation parameters are changed a bit.

```shell
[2022-02-22|10:31:21.029] attempt to set new parameter value; key: totalsupplyrange, value: "minTotalSupply:"100" maxTotalSupply:"1000000000000000" " module=x/params 
    simulate.go:300: error on block  1/2000, operation (16/37) from x/gov:
        failed to execute message; message index: 0: key: totalsupplyrange, value: "minTotalSupply:"100" maxTotalSupply:"1000000000000000" ", err: invalid character '1' after top-level value: failed to set parameter: invalid proposal content [cosmos/cosmos-sdk@v0.44.6/x/gov/keeper/proposal.go:24]
        Comment: unable to deliver tx
```

**To Reproduce**
Steps to reproduce the behavior:

Run `starport chain simulate` with the following modifications made to `app/simulation_test.go`:

```go
func BenchmarkSimulation(b *testing.B) {
	simapp.FlagEnabledValue = true
	simapp.FlagNumBlocksValue = 2000
	simapp.FlagBlockSizeValue = 100
	simapp.FlagCommitValue = true
	simapp.FlagPeriodValue = 10
	simapp.FlagVerboseValue = true
```

**Please provide output of commands**
 - `spnd version`: latest-2c5f4556
 - `go version`: go version go1.17.7 linux/amd64
 - `uname -a`: Linux cozart-dev 5.16.10-arch1-1 ... x86_64 GNU/Linux

--- a/x/campaign/module_simulation.go
+++ b/x/campaign/module_simulation.go
@@ -56,7 +56,10 @@ func (am AppModule) RandomizedParams(r *rand.Rand) []simtypes.ParamChange {
 	campaignParams := sample.CampaignParams()
 	return []simtypes.ParamChange{
 		simulation.NewSimParamChange(types.ModuleName, string(types.ParamStoreKeyTotalSupplyRange), func(r *rand.Rand) string {
-			return fmt.Sprintf("\"%v\"", campaignParams.TotalSupplyRange.String())
+			return fmt.Sprintf(
+				"{\"minTotalSupply\":\"%v\",\"maxTotalSupply\":\"%v\"}",
+				campaignParams.TotalSupplyRange.MinTotalSupply,
+				campaignParams.TotalSupplyRange.MaxTotalSupply)
 		}),
 	}
 }

